usecases.md

```diff
@@ -140,6 +140,74 @@ The table below shows the attributes for three different examples. We assume all
 | Site | Sensor X (America/Denver) | Power Plant Y (Etc/GMT+7) | Aggregate Z (America/Phoenix) |
 
 
+### Probabilistic forecasts
+{: .anchor}
+
+A *probabilistic forecast* in the Solar Forecast Arbiter is one or more
+[forecast evaluation time series](#forecastattrs) that describe part or all of a probability
+distribution.
+
+Let us consider a few simple examples. A probabilistic forecast may be the probability that
+generation is less than 10 MW. This probabilistic forecast is described by a
+single value (the probability) at each [forecast data point](#forecastdatapoint).
+This probability is traditionally known as a *quantile*. Another
```

Comment:
Its a `percentile` - quantile implies an interval in an equal-probability discretization of the range (and less commonly, the domain) of the CDF.